### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dreamfisher-nux**
I want to reserve more GUIDs for a diplomacy overhaul mod

### CODE REVIEW

**Review by TerraLion — COMMENTED** [review]
ein Schönbaumod

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@
 | Sir Ludicrous   | 2.100.120.000 | 2.100.199.999 |
 | Phloup          | 2.122.330.000 | 2.122.339.999 |
 | Maliatchi       | 2.122.340.000 | 2.122.340.999 |
-| Dreamfisher       | 2.122.350.000 | 2.122.350.199 |
+| Dreamfisher       | 2.122.350.000 | 2.122.350.999 |
```
[the remark this answers is not in the document]
Comment (taubenangriff) [reply README.md:46]: Hey, if you want to submit your guidrange, please create your own PullRequest. Thanks in advance!

[the remark this answers is not in the document]
Comment (TerraLion) [reply README.md:46]: thanks, sorry, I have problems to create at time. But i will learn

**Comment by TerraLion** [comment]
Gesendet: Mittwoch, 28. September 2022 um 12:12 Uhr
Von: "taubenangriff" ***@***.***>
An: "anno-mods/GuidRanges" ***@***.***>
Cc: "TerraLion" ***@***.***>, "Comment" ***@***.***>
Betreff: Re: [anno-mods/GuidRanges] Update README.md (PR #12)


 

@taubenangriff commented on this pull request.


In README.md:

> @@ -42,7 +42,7 @@
 | Sir Ludicrous   | 2.100.120.000 | 2.100.199.999 |
 | Phloup          | 2.122.330.000 | 2.122.339.999 |
 | Maliatchi       | 2.122.340.000 | 2.122.340.999 |
-| Dreamfisher       | 2.122.350.000 | 2.122.350.199 |
+| Dreamfisher       | 2.122.350.000 | 2.122.350.999 |



Hey, if you want to submit your guidrange, please create your own PullRequest. Thanks in advance!

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.

thanks for answer,  I have no practise in github and make an mistake, and in my english 

TerraLionMessage ID: ***@***.***>

**Pull request merged**